doc/feature/lineEvents.md

```diff
@@ -0,0 +1,59 @@
+Line events
+===========
+
+At the moment, Fivox works only with point events, which is not enough in the
+cases where more precision is needed, e.g. LFP computation on neuron structures
```

Comment:
This is not an just precision issue, it can completely change the results for some regions by magnitudes ( long lines projecting completely onto point ). It increases the correctness ( precision has a different meaning ).